examples/file-feed/complete/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "file-feed",
+  "version": "0.0.0",
+  "bin": "src/ff-cli/bin.js",
+  "devDependencies": {
+    "buffer": "^5.0.2",
```

Comment:
I'm not sure 100% either anymore. It was needed to force webpack to use the latest version (instead 3.x) to fix some of the errors we were seeing with browserify-zlib, but those might be gone now with the updated, correct browserify-zlib-next. I'll verify if this is still needed and remove it if not.